src/app/containers/IndexPageSection/index.jsx

```diff
@@ -95,8 +95,8 @@ const parentGridColumns = {
   group5: 8,
 };
 
-const renderPromos = (items, isFirstSection, dir) => {
-  const rows = getRows(items, isFirstSection);
+const renderPromos = ({ items, isFirstSection, dir, showAllRows }) => {
```

Comment:
Could we rename this prop to `showAllPromos`?

Reply:
Done.